.github/workflows/docker.yaml

```
@@ -0,0 +1,30 @@
+name: Build and push a new docker image based on project release tag.
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: 'Get Previous tag'
+        id: previoustag
+        uses: actions-ecosystem/action-get-latest-tag@v1
+        with:
+          fallback: 1.0.0 # Optional fallback tag to use when no tag can be found
```

the fallback should be latest
```suggestion
          fallback: latest # Optional fallback tag to use when no tag can be found
```